tests/z80/linker_test_1/main.s

```diff
@@ -273,6 +274,10 @@ _loop:
 .printv 100
 .printt "\n"
 
+// BT main.o
```

Comment:
Does this mean that we are missing these two tests that check if a definition exists or not?